server_admin/topics/threat/brute-force.adoc

```diff
@@ -1,84 +1,102 @@
+====
+
+.Procedure
 
+To enable this protection:
+
+. Click the *Realm Settings* menu
+. Click the *Security Defenses* tab.
+. Click the *Brute Force Detection* tab.
++
 .Brute Force Detection
 image:{project_images}/brute-force.png[]
 
-There are 2 different configurations for brute force detection; permanent lockout and temporary lockout. Permanent lockout will disable a user's account after an attack is detected; the account will be disabled until an administrator renables it. Temporary lockout will disable a user's account for a time period after an attack is detected; the time period for which the account is disabled increases the longer the attack continues.
-
-NOTE: When user is temporarily locked and attempt to login, the default error message `Invalid username or password` is shown.
-This is the same error message as the message displayed when invalid username or invalid password is provided. This is by design as
-we do not want to reveal to the attacker that user is temporarily disabled.
+{project_name} can deploy permanent lockout and temporary lockout actions when it detects an attack. Permanent lockout disables a user account until an administrator re-enables it. Temporary lockout disables a user account for a specific period of time. The time period that the account is disabled increases as the attack continues.
 
-*Common Parameters*
+[NOTE]
 ====
-Max Login Failures::
-  Maximum number of login failures permitted. Default value is 30.
-Quick Login Check Milli Seconds::
-  Minimum time required between login attempts. Default is 1000.
-Minimum Quick Login Wait::
-  Minimum amount of time the user will be temporarily disabled if logins attempts are quicker than _Quick Login Check Milli Seconds_. Default is 1 minute.
+When a user is temporarily locked and attempts to log in, {project_name} displays the default `Invalid username or password` error message. This message is the same error message as the message displayed for an invalid username or invalid password to ensure the attacker is unaware the account is disabled.
 ====
 
-*Temporary Lockout Parameters*
-====
-Wait Increment::
-  Amount of time added to the time a user is temporarily disabled after each time _Max Login Failures_ is reached. Default is 1 minute.
-Max Wait::
-  The maximum amount of time for which a user will be temporarily disabled. Default is 15 minutes.
-Failure Reset Time::
-  Time after which the failure count will be reset; timer runs from the last failed login. Default is 12 hours.
-====
+*Common Parameters*
+
+|===
+|Name |Description |Default
+
+|Max Login Failures
+|The maximum number of login failures.
+|30 failures.
 
-*Permanent Lockout Algorithm*
+|Quick Login Check Milliseconds
+|The minimum time between login attempts.
+|1000 milliseconds.
+
+|Minimum Quick Login Wait
+|The minimum time the user is disabled when login attempts are quicker than _Quick Login Check Milliseconds_.
+|1 minute.
+
+|===
+
```

Comment:
Are lines 62 to 80 intentionally moved later? It's difficult for me to see without merging and building to document.

Reply:
yes, they are. I wanted the temporary lockout and permanent lockout details in their own section. Before *Temporary Lockout Algorithm* and *Temporary Lockout Parameters* were in separate sections. Now they are together.